scalafix-tests/input/src/main/scala/test/disableSyntax/DisableSyntaxNoValPatterns.scala

```diff
@@ -0,0 +1,14 @@
+/*
+rules = DisableSyntax
+DisableSyntax.noValPatterns = true
+*/
+package test.disableSyntax
+
+case object DisableSyntaxNoValPatterns {
+  val Right(notFound) = 1.asInstanceOf[Either[String, String]] // assert: DisableSyntax.noValPatterns
```

Comment:
Can we add a test case in the basic DisableSyntax to check that this error is not reported unless `DisableSyntax.noValPatterns = true`?

Reply:
I have added the same line of code in DisableSyntaxBase.